src/quiz/quiz.stories.tsx

```diff
@@ -90,8 +91,105 @@ const QuizWithValidation = () => {
 	const { questions, validateAnswers, correctAnswerCount } = useQuiz({
 		initialQuestions,
 		validationMessages: {
-			correct: "Correct",
-			incorrect: "Incorrect",
+			correct: "Correct.",
+			incorrect: "Incorrect.",
+		},
+	});
+	const [disabled, setDisabled] = useState(false);
+
+	const handleSubmit = () => {
+		validateAnswers();
+		setDisabled(true);
+	};
+
+	return (
+		<div>
+			{correctAnswerCount && <p>Correct answers: {correctAnswerCount}</p>}
```

Comment:
```suggestion
			{!!correctAnswerCount && <p>Correct answers: {correctAnswerCount}</p>}
```
Nit, but this is one of the rare situations where this pattern fails. Since `0` is falsy, the whole thing evaluates to `0` and that's what gets rendered.